### PR TITLE
TW 904 Settings - switch security warnings off or on

### PR DESCRIPTION
**Pull request by HeatherStenson**

### CODE REVIEW

File: src/pages/docs/sending-requests/responses.md [hunk 1]
```diff
@@ -166,6 +166,9 @@ If it finds any potential security risks, Postman adds the number of warnings to
 
 <img alt="Security tab showing one warning" src="https://assets.postman.com/postman-docs/v10/api-response-security-tab-v10.jpg" width="900px"/>
 
+> You can hide the number of security warnings that Postman displays in the **Security** tab if this number is not important to you. In the Postman header, select the settings icon
+<img alt="Settings icon" src="https://assets.postman.com/postman-docs/icon-settings-v9.jpg#icon" width="16px">, then select **Settings > General** and turn off **Security Warnings count**. While the number of warnings will no longer display next to the **Security** tab, you can still select this tab to inspect any warnings.
```
Comment (mark-anthony-postman) [inline src/pages/docs/sending-requests/responses.md:170]: The note isn't rendering correctly- removing the line break before the icon should fix it

**Review by mark-anthony-postman — APPROVED** [review]
Needs one minor fix, otherwise looks great! 🌟

File: src/pages/docs/sending-requests/responses.md (hunk 1 above)
Comment (HeatherStenson) [reply src/pages/docs/sending-requests/responses.md]: Oh boy, this was a good one 😅 Thanks Mark!

**Review by savageCoder55 — CHANGES_REQUESTED** [review]
Small change requested.

File: src/pages/docs/sending-requests/responses.md [hunk 2]
```diff
@@ -166,6 +166,8 @@ If it finds any potential security risks, Postman adds the number of warnings to
 
 <img alt="Security tab showing one warning" src="https://assets.postman.com/postman-docs/v10/api-response-security-tab-v10.jpg" width="900px"/>
 
+> You can hide the number of security warnings that Postman displays in the **Security** tab if this number is not important to you. In the Postman header, select the settings icon <img alt="Settings icon" src="https://assets.postman.com/postman-docs/icon-settings-v9.jpg#icon" width="16px">, then select **Settings > General** and turn off **Security Warnings count**. While the number of warnings will no longer display next to the **Security** tab, you can still select this tab to inspect any warnings.
```
[the remark this answers is not in the document]
Comment (rishabjain22) [reply src/pages/docs/sending-requests/responses.md:169]: @savageCoder55 @HeatherStenson Yes, this functionality is only rolled out to 5% of non-Enterprise users.

**Review by claireriley — APPROVED** [review]
LGTM! ⭐

**Comment by jkonrath-postman** [comment]
This got shipped at some point and a different PR changed the settings, so this isn't needed anymore.

**Pull request closed without merging**